leaderboard/utils/final_score_table.py

```diff
@@ -1,8 +1,23 @@
 import pandas as pd
+import os
 
-from leaderboard.constants import scoreWeights
 from leaderboard.constants import contribTypes
 
+# get all the score weights from .env file
+score_wt_T1S1 = float(os.environ.get("T1S1", 1))
+score_wt_T1S2 = float(os.environ.get("T1S2", 1))
+score_wt_T2S1 = float(os.environ.get("T2S1", 1))
+score_wt_T2S2 = float(os.environ.get("T2S2", 1))
+score_wt_T2S3 = float(os.environ.get("T2S3", 1))
+score_wt_T2S4 = float(os.environ.get("T2S4", 1))
+score_wt_T3S1 = float(os.environ.get("T3S1", 1))
+score_wt_T3S2 = float(os.environ.get("T3S2", 1))
+score_wt_T4S1 = float(os.environ.get("T4S1", 1))
+score_wt_T4S2 = float(os.environ.get("T4S2", 1))
+score_wt_T4S3 = float(os.environ.get("T4S3", 1))
+score_wt_T4S4 = float(os.environ.get("T4S4", 1))
+score_wt_T5S1 = float(os.environ.get("T5S1", 1))
+
```

Comment:
These actually doesn't have to come through an env file. Can you get it from a `config.py`? These are the weights needed by algorithm itself and not a part of env. We can have an ability of environment override later if needed, but I don't think we need it.

Reply:
done

Reply:
done